src/frontend/src/session/cursor_manager.rs

```diff
@@ -770,33 +818,116 @@ impl SubscriptionCursor {
                 }
             })
             .collect::<Vec<_>>();
-        let output_col_idx_with_out_hidden = output_col_idx
+        let max_split_range_gap = context.session_ctx().config().max_split_range_gap() as u64;
```

Comment:
Why do we need this? I think we only have one range here.

Reply:
The call to the split_to_scan_ranges method requires that, here, he won't work